### PR TITLE
Change mode attribute to array in Measurements

### PR DESCRIPTION
**Pull request by tallysmartins**

### CODE REVIEW

File: lib/elixir_bench/benchmarks/measurement.ex [hunk 1]
```diff
@@ -40,10 +39,23 @@ defmodule ElixirBench.Benchmarks.Measurement do
...
+  end
+
+  defp set_mode(changeset, attrs) do
+    mode = get_in(attrs, ["mode"])
+
+    if is_integer(mode) or is_float(mode) do
+      change(changeset, mode: [mode])
+    else
+      changeset
+    end
```
Comment (PragTob) [inline lib/elixir_bench/benchmarks/measurement.ex]: we're not testing this functionality anywhere, are we?

Comment (tallysmartins) [reply lib/elixir_bench/benchmarks/measurement.ex]: tests added

File: test/elixir_bench/benchmarks/measurement_test.exs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+               :median,
+               :minimum,
+               :percentiles,
+               :run_times,
+               :sample_size,
+               :std_dev,
+               :std_dev_ips,
+               :std_dev_ratio
+             ] = Map.keys(changeset.changes)
+    end
```
Comment (PragTob) [inline test/elixir_bench/benchmarks/measurement_test.exs]: not sure we need this test, we more or less just repeat the list of fields. We should be fine without it I think :)

Comment (tallysmartins) [reply test/elixir_bench/benchmarks/measurement_test.exs]: it doesn't cast the `:mode` attribute, the only field that is not required. But I agree that this is kind of a weak test, this case is someway covered in the next test cases where I do `mode = nil`. Will remove this.

**Pull request merged**